tools/generate-pdfs.sh

```diff
@@ -18,26 +18,32 @@ set -ex
 
 function generate_pdfs_from_md() {
 	inputMdFile=$1
+
 	if ! [ -f "$inputMdFile" ]; then
 		echo "**** WARNING! $inputMdFile does not exist. Please enter valid file path."
 		echo "**** WARNING! Please make sure to run tools/generate-intrinsics-specs.sh before building the PDFs."
 		exit 1
 	fi
 
 	outputPdfFile=$2
-	configYamlFile=$3
-	headingLineNum=$(awk '/<!---END_OF_HTML_HEADER--->/ { print NR; exit }' $inputMdFile)
+	geometryForIntrinsics='\\newgeometry{landscape,top=3.7cm,bottom=2.7cm,left=1cm,right=1cm,headsep=1.5cm,footskip=.5cm}'
 
-	tail -n +$headingLineNum $inputMdFile | \
-	pandoc --template=tools/acle_template.tex --metadata-file=$configYamlFile -o $outputPdfFile
+	# This line replaces the ToC declaration in the md files with a blank space.
+	# ":a;N;$!ba;" is at the start so sed could recognise newline.
+	# The rest is a regular expression.
+	# The second replacement string is being used to format the "List of Intrinsics"
+	# sections, which contain large longtables.
+	sed -u ':a;N;$!ba;s/\*\sTOC\n{*{:toc}}*//' $inputMdFile | \
+	sed -u "s/<!--latex_geometry_conf-->/${geometryForIntrinsics}/" | \
```

Comment:
Please replace `<!--latex_geometry_conf-->` with a variable. Also, why are you using `${geometryForIntrinsics}` instead of just `$geometryForIntrinsics`? If there is no technical reason for using `${geometryForIntrinsics}`, please use `$geometryForIntrinsics` for the sake of uniformity with the other variables in the code.

Reply:
I think I left that in after debugging.